web/index.html

```diff
@@ -63,6 +63,16 @@
 
   <link type="text/css" rel="Stylesheet" href="build/wv.css?v=@BUILD_NONCE@">
   <script type="text/javascript" src="build/wv.js?v=@BUILD_NONCE@"></script>
+
+  <link rel="manifest" href="/manifest.webmanifest">
+  <meta name="theme-color" content="#0b3d91"/>
+  <script>
```

Comment:
Does this need to be in a <script> tag or can we just include this JavaScript in our `main.js` file?

Reply:
We could try putting it in the `main.js`, but I think we'll need to make a change to the code that registers the service worker's scope, i.e. the highest directory from which it's working from despite the location worker itself.

```javascript
navigator.serviceWorker.register('js/service-worker.js', {
  scope: '/'
});
```

Taken and modified from this page:  https://developers.google.com/web/ilt/pwa/introduction-to-service-worker#registration_and_scope.  

Also:
> If you want the service worker to control higher pages e.g. /app (without the trailing slash) you can indeed change the scope option, but you'll also need to set the `Service-Worker-Allowed` HTTP Header in your server config for the request serving the service worker script.

I don't know if any of this is really necessary if we're not planning on caching anything, but if we do decide to cache the `wv.js` or anything else in a directory above the service worker, we'll may need to keep this all in mind.

Reply:
There is no `js` folder in the final build directory structure, would it not just live in `web/js` alongside `main.js` and the pathing would be the same?